### ISSUE TITLE
fill-opacity (in groups) not supported?

### ISSUE THREAD

**Issue by sir-boformer** [dnfield/flutter_svg#39]
```svg
<?xml version="1.0" encoding="UTF-8"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->
<svg width="96" height="96" version="1.1" viewBox="0 0 96 96" xmlns="http://www.w3.org/2000/svg" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
 <g transform="translate(0 -260.93)" fill="#ff7f00">
  <path d="m82 340.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".31373"/>
  <path d="m66 340.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m66 324.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".31373"/>
  <path d="m82 324.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m50 324.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m50 340.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".31373"/>
  <path d="m50 308.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".15686"/>
  <path d="m66 308.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".31373"/>
  <path d="m82 308.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m34 340.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m18 340.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".15686"/>
  <path d="m18 324.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".15686"/>
  <path d="m2 340.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".078431"/>
  <path d="m2 308.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".078431"/>
  <path d="m34 308.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m82 292.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".15686"/>
  <path d="m66 292.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m82 276.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".23529"/>
  <path d="m50 276.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".15686"/>
  <path d="m82 260.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".078431"/>
  <path d="m18 276.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".078431"/>
  <path d="m34 292.93h10c1.108 0 2 0.892 2 2v10c0 1.108-0.892 2-2 2h-10c-1.108 0-2-0.892-2-2v-10c0-1.108 0.892-2 2-2z" fill-opacity=".15686"/>
 </g>
</svg>
```

In Chrome:
![grafik](https://user-images.githubusercontent.com/22688477/43460680-a4d06f84-94d1-11e8-89c9-2e868e518ecf.png)

In Flutter:
![grafik](https://user-images.githubusercontent.com/22688477/43460854-2e2e9210-94d2-11e8-9109-a64525c40b8d.png)

**Comment by sir-boformer** [comment]
Modifying the svg like this produces the correct output:
```svg
 <g transform="translate(0 -260.93)">
  <path d="..." fill="#fff" fill-opacity=".31373"/>
  ...
 </g>
```

**Comment by dnfield** [comment]
Bug seems to be how fill-opacity is applied when fill is inherited.  Will take a look.

**Comment by dnfield** [comment]
Yup, I'm ignoring opacity when the fill color is inherited.  Should be able to sort this out pretty quickly...

**Comment by dnfield** [comment]
Fixed in 71a7a26 - will be v0.5.2

**Comment by dnfield** [comment]
Have a fix for fill, looks like stroke would suffer from a similar issue so trying to resolve that as well.

**Comment by dnfield** [comment]
FYI - this caused a regression that was fixed in what is now v0.5.5, which should still support this and more inheritable properties better for fills/strokes.

**Comment by sir-boformer** [comment]
Just tested this, works perfectly!
